### PR TITLE
Small fixes for rusty hooks

### PR DESCRIPTION
**Pull request by geigerzaehler**
Another set of small fixes for rusty hooks. See the commit descriptions for details.

### CODE REVIEW

File: ci/src/bin/buildkite-pre-checkout-hook.rs [hunk 1]
```diff
@@ -55,15 +52,10 @@ fn decrypt_repo_secrets(cfg: &Config) -> Result<(), Error> {
...
     info!("Adjusting checkout path ownership");
 
-    let mut chown = Command::sudo();
-
-    chown
+    Command::sudo()
         .args(&["chown", "-R", "buildkite-agent"])
-        .arg(&cfg.checkout_path);
-
-    if let GoogleApplicationCredentials::Json(path) = &cfg.google_application_credentials {
```
Comment (kim) [inline ci/src/bin/buildkite-pre-checkout-hook.rs:64]: Uh, yeah. But I guess `sops` needs it instead

File: ci/src/cmd.rs [hunk 2]
```diff
@@ -136,7 +136,11 @@ pub trait CommandExt {
 }
 
 impl CommandExt for Command {
-    fn sudo() -> Command { Command::new("sudo") }
+    fn sudo() -> Command {
+        let mut cmd = Command::new("sudo");
+        cmd.arg("--non-interactive");
```
Comment (kim) [inline ci/src/cmd.rs:141]: Smart move :eyeglasses:

File: ci/src/bin/buildkite-pre-checkout-hook.rs (hunk 1 above)
Comment (geigerzaehler) [reply ci/src/bin/buildkite-pre-checkout-hook.rs:64]: But this command only runs `sudo chown ...`. I’ve added this to the `sops` command in https://github.com/radicle-dev/infra/pull/106/commits/dabe364da447a2a50517c108a17bc4ddeb85e936

Comment (kim) [reply ci/src/bin/buildkite-pre-checkout-hook.rs:64]: :+1:

**Pull request merged**